project_generator/templates/uvision4.uvproj.tmpl

```diff
@@ -377,8 +377,7 @@
             <ScatterFile>{{linker_file}}</ScatterFile>
             <IncludeLibs></IncludeLibs>
             <IncludeLibsPath></IncludeLibsPath>
-            <Misc>{% for lib in source_files_lib %} {{lib}} {% endfor %}
-                  {% for obj in source_files_obj %} {{obj}} {% endfor %}</Misc>
+            <Misc></Misc>
```

Comment:
yes - they're in the project tree and with .ar and .a and .lib you can use the dialogs to choose which object files are used in the archive :)